docs/products/mysql/howto/connect-from-mysqlx.rst

```diff
@@ -0,0 +1,59 @@
+Connect to MySQL using MySQLx with Python
```

Comment:
The title says "to", but the file name is "from", let's make it same

Reply:
I'm not sure this is correct or not, maybe @TibsAtWork can help.

Reply:
I think I got it now, you mean file names. Replaced :)